server/events/plan_command_runner.go

```diff
@@ -229,6 +229,12 @@ func (p *PlanCommandRunner) run(ctx *command.Context, cmd *CommentCommand) {
 				if err := p.commitStatusUpdater.UpdateCombinedCount(baseRepo, pull, models.SuccessCommitStatus, command.Plan, 0, 0); err != nil {
 					ctx.Log.Warn("unable to update commit status: %s", err)
 				}
+				if err := p.commitStatusUpdater.UpdateCombinedCount(baseRepo, pull, models.SuccessCommitStatus, command.PolicyCheck, 0, 0); err != nil {
+					ctx.Log.Warn("unable to update commit status: %s", err)
+				}
+				if err := p.commitStatusUpdater.UpdateCombinedCount(baseRepo, pull, models.SuccessCommitStatus, command.Apply, 0, 0); err != nil {
```

Comment:
Can we make sure this doesn't affect issues like https://github.com/runatlantis/atlantis/issues/3725 and cause a regression?

Reply:
Not sure I understand what you are asking here - this part of the code just replicates what autoplan does. Is autoplan broken wrt #3725?

Reply:
I wanted to clarify that we are considering all possible states and have tested for them. https://github.com/runatlantis/atlantis/pull/3747 and https://github.com/runatlantis/atlantis/pull/3378 are better context around commit statuses